src/classes/UTIL_CustomSettingsFacade.cls

```diff
@@ -35,6 +35,10 @@
 * Needs to be "without sharing" so that the Install Script can run it.
 */
 public without sharing class UTIL_CustomSettingsFacade {
+    
+   	public static boolean userCanEditDataImportSettings {get {
```

Comment:
@cdcarter any reason this is in the facade? feels like it should be directly in the controller since it's only used there.